src/index.ts

```diff
@@ -842,7 +841,11 @@ export class Bigtable {
           currentRetryAttempt: 0,
           noResponseRetries: 0,
           objectMode: true,
-          shouldRetryFn: shouldRetryRequest,
+          // TODO: re-enable shouldRetryFn when the retry
+          // logic is fixed in gax / retry-request
+          shouldRetryFn: (_: any) => {
+            return false;
+          },
```

Comment:
Why does this have to happen here? don't you overwrite there in mutateRows & readRows methods?